glue_solar/core/core.py

```diff
@@ -0,0 +1,70 @@
+"""
+A reader for sunpy map data.
+"""
+import os
+
+from pathlib import Path
+
+from qtpy import QtWidgets
+
+from astropy.io import fits
+from astropy.wcs import WCS
+
+import sunpy.map
+from sunpy.map.mapbase import GenericMap  # isort:skip
```

Comment:
Why was this isort skiped?

Reply:
Hmm... I think I copied this line from elsewhere. Let me try removing it and see if anything bad will happen.